extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/OidcDevServicesBuildStep.java

```diff
@@ -0,0 +1,70 @@
+package io.quarkus.oidc.deployment.devservices;
+
+import java.util.HashMap;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.jboss.logging.Logger;
+
+import io.quarkus.deployment.IsNormal;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.BuildSteps;
+import io.quarkus.deployment.builditem.DockerStatusBuildItem;
+import io.quarkus.deployment.dev.devservices.GlobalDevServicesConfig;
+import io.quarkus.devservices.keycloak.KeycloakDevServicesConfig;
+import io.quarkus.devservices.keycloak.KeycloakDevServicesRequiredBuildItem;
+import io.quarkus.oidc.deployment.OidcBuildStep;
+import io.quarkus.runtime.configuration.ConfigUtils;
+
+@BuildSteps(onlyIfNot = IsNormal.class, onlyIf = { OidcBuildStep.IsEnabled.class, GlobalDevServicesConfig.Enabled.class })
+public class OidcDevServicesBuildStep {
```

Comment:
```suggestion
public class KeycloakDevServiceRequiredBuildStep {
```

in `quarkus-oidc`, the code which lives directly in the `.devservices.` package, is meant to support not only Keycloak in devmode, so I propose rename this build step, and move further down in the `devservices.keycloak` package. And probably makes sense to do he same  for other extensions for consistency

Reply:
I didn't think of that, thanks.